README.md

```diff
@@ -1,10 +1,376 @@
+  const [name, setName] = useState('unknown');
+
+  return (
+    <>
+      <div>
+        <a href="https://vite.dev" target="_blank">
+          <img src={viteLogo} className="logo" alt="Vite logo" />
+        </a>
+        <a href="https://react.dev" target="_blank">
+          <img src={reactLogo} className="logo react" alt="React logo" />
+        </a>
+      </div>
+      <h1>Vite + React</h1>
+      <div className="card">
+        <button
+          onClick={() => setCount((count) => count + 1)}
+          aria-label="increment"
+        >
+          count is {count}
+        </button>
+        <p>
+          Edit <code>src/App.tsx</code> and save to test HMR
+        </p>
+      </div>
+      <div className="card">
+        <button
+          onClick={() => {
+            fetch('/api/')
+              .then((res) => res.json() as Promise<{ name: string }>)
+              .then((data) => setName(data.name));
+          }}
+          aria-label="get-name"
+        >
+          Name from API is: {name}
+        </button>
+        <p>
+          Edit <code>api/index.ts</code> to change the name
+        </p>
+      </div>
+      <p className="read-the-docs">
+        Click on the Vite and React logos to learn more
+      </p>
+    </>
+  );
+}
+
+export default App;
 ```
+
+Now, if you click the button, it will display 'Name from API is: Cloudflare'.
+
+Try incrementing the counter and then changing the `name` that is returned in `api/index.ts`.
+If you click the button again, it will display the new value while preserving the counter state.
+
+### Build your application
+
+Run `vite build` to build your application.
+
+If you inspect the `dist` directory, you will see that it contains two subdirectories: `client` and `api`.
+The `api` directory contains your Worker code and the output `wrangler.json` configuration.
+
+### Preview your application
+
+Run `vite preview` to validate that your application runs as expected.
+This command will run your build output locally in the Workers runtime.
+
+### Deploy to Cloudflare
+
+Run `wrangler deploy` to deploy your application to Cloudflare.
+This command will automatically use the output `wrangler.json` that was included in the build output.
+
+### Next steps
+
+In this tutorial, we created an SPA that could be deployed using Workers Assets.
+We then added an API Worker that could be accessed from the front-end code.
+Possible next steps include:
+
+- Adding a binding to another Cloudflare service such as a [KV namespace](https://developers.cloudflare.com/kv/) or [D1 database](https://developers.cloudflare.com/d1/)
+- Expanding the API to include additional routes
+- Using a library, such as [tRPC](https://trpc.io/) or [Hono](https://hono.dev/), in your API Worker
+
+## API
+
+### `cloudflare`
+
+The `cloudflare` plugin should be included in the Vite `plugins` array:
+
+```ts
+// vite.config.ts
+
+import { defineConfig } from 'vite';
+import { cloudflare } from '@cloudflare/vite-plugin';
```

Comment:
are we moving away from the default import?

Reply:
See other comment about possibly importing a different package. As far as I remember, we never used a default export.